types/elasticsearch/elasticsearch-tests.ts

```diff
@@ -37,6 +37,14 @@ client.search({
 }, (error) => {
 });
 
+client.search({
+  q: 'pants'
```

Comment:
Should the query match the structure in the docs with a `_name` element?

Reply:
Yeah it probably should. I was mimicking the current test style where the query doesn't really relate to what's being tested anyway (since queries currently are not typed at all). But i'll make it better.